# Define the goal
string trajectory_name
float64 ee_error_th  # Specify error threshold
---
# Define the result
int32 error_code
int32 SUCCESSFUL = 0
int32 INVALID_GOAL = -1
int32 INVALID_JOINTS = -2
int32 OLD_HEADER_TIMESTAMP = -3
int32 PATH_TOLERANCE_VIOLATED = -4
int32 GOAL_TOLERANCE_VIOLATED = -5
---
# Define a feedback message
float64 percent_complete
